editions/tw5.com/tiddlers/nodejs/Serving TW5 from Android.tid

```diff
@@ -1,13 +1,46 @@
-created: 20160602043529506
-modified: 20160602043531313
-tags: [[TiddlyWiki on Node.js]]
+caption: Node.js on Termux
+created: 20200501120322327
+delivery: App with DIY steps
+description: Using Node.js to serve/create flatfile wikis
+method: sync
+modified: 20200501120801899
+tags: Saving [[TiddlyWiki on Node.js]] Android
 title: Serving TW5 from Android
 type: text/vnd.tiddlywiki
 
-[[Termux|https://termux.com/]] is an open source Android application that combines a Linux system and a terminal.
+[[Termux|https://termux.com/]] is and opensource android application providing limited unix environment enabling users to install [[Node.js]] and npm modules in android. Users can install and run [[TiddlyWiki on Node.js]] using [[Termux|https://termux.com/]].
```

Comment:
I think "and opensource" is a misprint here? Open source is usually two words in English. And Unix should be capitalised.

Reply:
Corrected